modules/jt-extending-tekton-using-custom-tasks-and-scripts.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-extending-tekton-using-custom-tasks-and-scripts_{context}"]
+= Extending Tekton using custom tasks and scripts
```

Comment:
= Extending Tekton using custom tasks and scripts --> = Extending Tekton capabilities using custom tasks and scripts